server/plugin/plugin.go

```diff
@@ -81,6 +87,48 @@ func (p *MatterpollPlugin) checkServerVersion() error {
 	return nil
 }
 
+// ensureBotAccount checks if the bot account exists and creates him if doesn't
+func (p *MatterpollPlugin) ensureBotAccount() error {
+	botUserID, err := p.Store.Bot().GetID()
+	if err != nil {
+		return errors.Wrap(err, "failed to get bot user from store")
+	}
+	if botUserID == "" {
+		bundlePath, err := p.API.GetBundlePath()
+		if err != nil {
+			return errors.Wrap(err, "failed to get bundle path")
+		}
+
+		profileImage, err := ioutil.ReadFile(filepath.Join(bundlePath, "assets", "logo_dark.png"))
+		if err != nil {
+			return errors.Wrap(err, "failed to read profile image")
+		}
+
+		bot := &model.Bot{
+			Username:    "matterpoll",
+			DisplayName: "Matterpoll",
+			Description: "Some description",
+		}
+
+		rbot, appErr := p.API.CreateBot(bot)
+		if appErr != nil {
+			return errors.Wrap(appErr, "failed to create bot account")
+		}
+		botUserID = rbot.UserId
+
+		if appErr := p.API.SetProfileImage(botUserID, profileImage); appErr != nil {
```

Comment:
I'm a bit unhappy with this. If `SetProfileImage` fails, the bot account will never have a profile image. But I also don't want to set the image on every startup.

Reply:
Hmm...I'm not too clear about that point.
If `SetProfileImage` fails, this func returns error, so plugin never be started. Bots without images will never be registered. Is this right?

Reply:
> Bots without images will never be registered. Is this right?

No. The account will be crated in L113.

Reply:
Yes, Bot account is created, but the account will not be registered to KVStore, because this func return error before calling L123. A zombie bot account still alive.

So I suppose that Bots without images will never be registered. I think that this is not problem for now.

Reply:
I will change the code to always set the profile image.